tutorials.code-workspace

```diff
@@ -1,7 +1,7 @@
 {
 	"folders": [
 		{
-			"path": "."
+			"path": "connecting-to-testnet"
```

Comment:
This is for your local workspace? Is this helpful for others to have this file `tutorials.code-worspace` in the repository or should we consider adding this file to the `.gitignore`?

Reply:
I like the .gitignore since workspaces are personal. wdyt?

Reply:
Yes, let's add it in this PR